examples/src/main/java/alluxio/cli/MiniBenchmark.java

```diff
@@ -46,94 +61,157 @@
 
   private MiniBenchmark() {} // prevent instantiation
 
+  /**
+   * Prints the usage.
+   */
   private static void usage() {
-    System.out.println("Usage:");
-    System.out.println("To run a mini benchmark to write or read a file.");
-    System.out.println(String
-        .format("java -cp %s %s <[READ, WRITE]> <fileSize> <iterations>",
-        RuntimeConstants.ALLUXIO_JAR, MiniBenchmark.class.getCanonicalName()));
+    new HelpFormatter().printHelp(
+        String.format("java -cp %s %s <[READ, WRITE]> <fileSize> <iterations>",
```

Comment:
update the usage with the latest options `-type`, `-fileSize`, `-iterations` and `-concurrency`?